qa/tasks/vstart_runner.py

```diff
@@ -360,7 +360,7 @@ def run(self, args, check_status=True, wait=True,
         else:
             # Sanity check that we've got a list of strings
             for arg in args:
-                if not isinstance(arg, basestring):
+                if not isinstance(arg, str):
```

Comment:
@kshtsk could you also add a commit. something like?

```diff
diff --git a/qa/tasks/vstart_runner.py b/qa/tasks/vstart_runner.py
index 87e781b65b..f640230f62 100644
--- a/qa/tasks/vstart_runner.py
+++ b/qa/tasks/vstart_runner.py
@@ -44,6 +44,7 @@ import time
 import sys
 import errno
 from unittest import suite, loader
+import six
 import unittest
 import platform
 from teuthology.orchestra.run import Raw, quote
@@ -176,6 +177,7 @@ class LocalRemoteProcess(object):
                 return

         out, err = self.subproc.communicate()
+        out, err = out.decode(), err.decode()
         self.stdout.write(out)
         self.stderr.write(err)

@@ -283,7 +285,7 @@ class LocalRemote(object):
     def _perform_checks_and_return_list_of_args(self, args, omit_sudo):
         # Since Python's shell simulation can only work when commands are
         # provided as a list of argumensts...
-        if isinstance(args, str) or isinstance(args, unicode):
+        if isinstance(args, str) or isinstance(args, six.text_type):
             args = args.split()

         # We'll let sudo be a part of command even omit flag says otherwise in
```

Reply:
maybe it is better to use, BytesIO with six.ensure_str?